packages/mui-base/src/Tooltip/Popup/TooltipPopup.types.ts

```diff
@@ -0,0 +1,49 @@
+import type { Side, VirtualElement } from '@floating-ui/react';
+import type { BaseUIComponentProps, GenericHTMLProps } from '../../utils/BaseUI.types';
+import type { TooltipPopupParameters } from './useTooltipPopup.types';
+import type { TooltipPopupRootOwnerState } from '../PopupRoot/TooltipPopupRoot.types';
+
+export interface TooltipPopupContextValue {
+  open: boolean;
+  side: Side;
+  alignment: 'start' | 'end' | 'center';
+  arrowRef: React.MutableRefObject<Element | null>;
+  arrowUncentered: boolean;
+  getArrowProps: (externalProps?: GenericHTMLProps) => GenericHTMLProps;
+}
+
+export type TooltipPopupOwnerState = {
+  open: boolean;
+  side: Side;
+  alignment: 'start' | 'end' | 'center';
+  instant: 'delay' | 'focus' | 'dismiss' | undefined;
+  entering: boolean;
+  exiting: boolean;
+};
+
+export interface TooltipPopupProps
```

Comment:
I'm not sure where it's coming from, but VSCode suggests `setMounted` as a Popup prop. It doesn't seem correct (`onMounted` would be better if we need to expose it)

Reply:
It was due to extending `TooltipPopupParameters` (the hook's low-level params) without omitting them. I've updated the type